jib-core/src/test/java/com/google/cloud/tools/jib/image/ReproducibleLayerBuilderTest.java

```diff
@@ -236,7 +258,7 @@ public void testBuild_parentDirBehavior() throws IOException {
       // root (default folder permissions)
       TarArchiveEntry root = in.getNextTarEntry();
       assertThat(root.getMode()).isEqualTo(040755);
-      assertThat(root.getModTime().toInstant()).isEqualTo(Instant.ofEpochSecond(1));
```

Comment:
I've replaced the assertion with the helper `verifyThatModificationTimeIsReset`.  It asserts that the mod time is just "EPOCH".  The only change i did outside of the test code here is to set the PAX headers to 0 -- I guess I can set them to 1 (second after epoch) instead?  
Why is it important that it's specifically EPOCH+1 and not just EPOCH?

Reply:
> Why is it important that it's specifically EPOCH+1 and not just EPOCH?

#1079 was the issue.

It's interesting that clearing the PAX header makes `getLastModifiedDate()` return 0. What if you use the DEFAULT_MODIFICATION_TIME? I think the ordinary mod time may have precedence over the PAX header. If not, then it's a problem, because it has to epoch+1.

Reply:
Oh, `DEFAULT_MODIFICATION_TIME` is a static time, so looks like the PAX headers really interfere, right? Can you try calling `clearPaxTimeHeaders()` before `setModTime()` in the main code (two places)?

Reply:
Yes, the PAX headers do interfere because that's the only change I did and after that these tests failed because 1!=0.  I'll try and switch around and see what happens, and I'll also just the the PAX headers to 1 instead.

Reply:
The PAX headers are now set to 1 and the test helper asserts this.  It didn't seem to make any difference when I switched the ordering of the setModTime and clearPaxHeaders call, any regardless of that they should agree.